internal/statemachine/classic_test.go

```diff
@@ -4593,31 +4612,6 @@ func TestStateMachine_installPackages_fail(t *testing.T) {
 	err = stateMachine.installPackages()
 	asserter.AssertErrContains(err, "Error setting up /etc/resolv.conf")
 	helperBackupAndCopyResolvConf = helper.BackupAndCopyResolvConf
-
```

Comment:
With this removed we are not testing anymore that errors happening while diverting/undiverting are surfaced by  `runCmdsWithChrootSetup`. Would you mind checking at least one of the 3 diverts?

Reply:
@fhost I think this still remains to be addressed.

Reply:
Is this enough?